README.md

```diff
@@ -8,11 +8,30 @@
 [![Dependency Status](https://deps.rs/repo/github/rusqlite/rusqlite/status.svg)](https://deps.rs/repo/github/rusqlite/rusqlite)
 [![Discord Chat](https://img.shields.io/discord/927966344266256434.svg?logo=discord)](https://discord.gg/nFYfGPB8g4)
 
-Rusqlite is an ergonomic wrapper for using SQLite from Rust. It attempts to expose
-an interface similar to [rust-postgres](https://github.com/sfackler/rust-postgres).
```

Comment:
At least we can still stay that `rusqlite` API has been inspired from `rust-postgres` (like `ToSql` / `FromSql`). And, for me, this link is convenient when someone challenges `rusqlite` API (and when `rust-postgres` does the same).
Also, by analogy, https://www.pgcon.org/2014/schedule/events/736.en.html:
> SQLite can be thought of as a derivative of PostgreSQL. SQLite was originally written from PostgreSQL 6.5 documentation, and the SQLite developers still use PostgreSQL as a reference platform to verify that SQLite is working correctly.

Reply:
Hm, fair enough. In practice ISTM we're pretty far away already, and both of us are moving further apart...  I'll see if I can think of a better way to word it that reflects the history without implying anything more.

I want to avoid implying there's any compatibility or intent of compatibility or similarity between the two libraries. My hesitation is mostly that some users may see this and expect/hope that they will be able to use the similarities to enable use of SQLite for development and postgres for production, which will... cause them frustration if they try to do so with `rusqlite` and `rust-postgres`. IMO that situation is not good for anybody[^1], and really, I think it'd end up being very frustrating for the user, so I'd rather not encourage it at all.

Honestly users who want the compatibility will be way better served by using neither, and instead finding a database abstraction library that actually *intends* to provide compatibility across DBs (rather than hacking it together via clever `#[cfg()]`s or whatever), since there are several decent ones that do at this point.

[^1]: Well, for reasons I can't really place I feel like `rusqlite` fairs worse in that situation. Perhaps because we're not the one that is used in production, but maybe just because we're the one making the claim of similarity.